fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -413,7 +413,8 @@
 			new_sash = new /obj/item/clothing/under/bodysash()
 		if(JOB_PRISONER)
 			new_sash = new /obj/item/clothing/under/bodysash/prisoner()
-
+			var/obj/item/implant/tracking/tracking_implant = new /obj/item/implant/tracking/
+			tracking_implant.implant(equipping, null , TRUE)
```

Comment:
```suggestion
			var/obj/item/implant/tracking/tracking_implant = new /obj/item/implant/tracking
			tracking_implant.implant(equipping, null, TRUE)
```

you probably could also get away with only using new() as the variable already points to the implant but I'd have to double check if that will work